prog/vm/nexus.rb

```diff
@@ -588,6 +588,7 @@ def write_params_json
       end
       lb.remove_vm(vm)
     end
+    nap 5 unless vm.removed_from_lb_set?
```

Comment:
This looks odd to me (maybe it is correct, but it looks odd). Shouldn't we only do this check if the vm has/had a load balancer? If the vm never had a load balancer, doesn't this just nap 5 seconds until the semaphore is set manually?